src/Forms/ChooseEncoding.cs

```diff
@@ -12,6 +12,11 @@ public sealed partial class ChooseEncoding : Form
         private Encoding _encoding;
         private byte[] _fileBuffer;
 
+        public Encoding GetEncoding
+        {
+            get { return _encoding; }
+        }
+

```

Comment:
As it is no longer a method, perhaps, you could rename `GetEncoding` to `Encoding`?